docs/source/examples/function-calling.rst

```diff
@@ -89,11 +91,33 @@ The ``service.py`` file outlines the logic of the two required BentoML Services.
           },
       )
       class Llama:
+         # Declare the model as a class variable
+         hf_model = bentoml.models.HuggingFaceModel(MODEL_ID)
+
          def __init__(self) -> None:
          # Logic to initialize the model and tokenizer
          ...
 
-3. Next, use the ``@bentoml.service`` decorator to create another BentoML Service called ``ExchangeAssistant``. Different from the LLM, function calling does not require GPUs and can be run with a single CPU. Running them on separate instances also means you can scale them independently on BentoCloud later.
+   Within the class, :ref:`load the model from Hugging Face <load-models>` and define it as a class variable. The ``HuggingFaceModel`` method provides an efficient mechanism for loading AI models to accelerate model deployment, reducing image build time and cold start time.
```

Comment:
ig that this only applies to BentoCloud, otherwise, standalone deployment are still the same.

Reply:
Let me clarify this.